azurelinuxagent/common/conf.py

```diff
@@ -174,15 +174,23 @@ def enable_rdma(conf=__conf__):
 def enable_rdma_update(conf=__conf__):
     return conf.get_switch("OS.UpdateRdmaDriver", False)
 
+
 def enable_check_rdma_driver(conf=__conf__):
     return conf.get_switch("OS.CheckRdmaDriver", True)
 
+
 def get_logs_verbose(conf=__conf__):
     return conf.get_switch("Logs.Verbose", False)
 
+
 def get_logs_console(conf=__conf__):
     return conf.get_switch("Logs.Console", True)
 
+
+def get_logs_to_telemetry(conf=__conf__):
```

Comment:
is this something we want users to be able to change? (wouldn't think so) the conf file doesn't seem to be appropriate for this.